components/sup/src/main.rs

```diff
@@ -110,7 +110,7 @@ fn cli<'a, 'b>() -> App<'a, 'b> {
                 "The listen address of an initial peer (IP[:PORT])")
             (@arg PERMANENT_PEER: --("permanent-peer") -I "If this service is a permanent peer")
             (@arg RING: --ring -r +takes_value "Ring key name")
-            (@arg PKG_IDENT_OR_ARTIFACT: +required
```

Comment:
Did you see my (evil) hack to preserve the appearance of this being required without the flag? That'll be deleted soon enough, but no surprises for people in the meantime.